core/feed_merges.go

```diff
@@ -0,0 +1,27 @@
+package core
+
+import (
+	"github.com/textileio/textile-go/pb"
+)
+
+func (t *Textile) merge(block *pb.Block, opts feedItemOpts) (*pb.Merge, error) {
```

Comment:
Even though `Feed` does not return this type of block (or announces, flags, etc.), the `sub` command can, so I've added `FeedItems` for _all_ types.